wolfssl/wolfcrypt/settings.h

```diff
@@ -1971,6 +1956,13 @@ extern void uITRON4_free(void *p) ;
         #endif
     #endif
 #endif
+
+/* Verify that only one of the above multi-precision math libraries is enabled */
+#if (defined(WOLFSSL_SP_MATH_ALL) && \
+        (defined(USE_FAST_MATH) || defined(USE_INTEGER_HEAP_MATH))) || \
+    (defined(USE_FAST_MATH) && defined(USE_INTEGER_HEAP_MATH))
+    #error Cannot enable more than one multiple precision math library!
```

Comment:
missed an opportunity for a really fun preprocessor construction here (this actually works!)
```
#if defined(WOLFSSL_SP_MATH_ALL) + defined(USE_FAST_MATH) + defined(USE_INTEGER_HEAP_MATH) > 1
#error nope
#endif
```
not seriously suggesting... not even sure it's portable.

Reply:
Yours is much more elegant, but I cannot guarantee portability. I think they would have to =1 and that is just not a guarantee.